mappings/net/minecraft/server/world/ServerWorld.mapping

```diff
@@ -84,6 +100,10 @@ CLASS net/minecraft/class_3218 net/minecraft/server/world/ServerWorld
 	METHOD method_14195 resetWeather ()V
 	METHOD method_14197 resetIdleTimeout ()V
 	METHOD method_14199 spawnParticles (Lnet/minecraft/class_2394;DDDIDDDD)I
+		COMMENT Spawns a particle visible to nearby players. Returns the number of players
+		COMMENT the particle packet was sent to.
```

Comment:
This should be documented using a `@return` tag.

Reply:
```suggestion
		COMMENT Spawns a particle visible to nearby players.
		COMMENT
		COMMENT @return the number of players the particle packet was sent to
```